webpack.config.js

```diff
@@ -7,7 +7,7 @@ const HtmlPlugin = require('html-webpack-plugin')
 const FriendlyErrorsPlugin = require('friendly-errors-webpack-plugin')
 const StylelintPlugin = require('stylelint-webpack-plugin')
 const ManifestPlugin = require('webpack-manifest-plugin')
-const CleanPlugin = require('clean-webpack-plugin')
+const CleanWebpackPlugin = require('clean-webpack-plugin')
```

Comment:
Could you please rename it back to the original name? I prefer not to use 'webpack' in plugins' names since it's clear that they're plugins for webpack 😃